params.go

```diff
@@ -45,6 +46,18 @@ type StringSliceParam struct {
 	value []string
 }
 
+type URLParam struct {
+	baseParam
+	def   string
```

Comment:
This is how onion does Durations. Maybe the better move here is to add support for URLs to onion?
https://github.com/fzerorubigd/onion/blob/v2/onion.go#L351